Test.C

```diff
@@ -117,56 +115,110 @@ int main(){
-        //edit encrypt structure effectiveness 
+        //present attack
+        //print attack details and affects of attack
+        //edit encrypt structure effectiveness
 
-        //spend credits interactive menu 
+        //spend credits interactive menu
+        printf("|-------------------------------- Credit Store --------------------------------|\n");
 
         //display available equipment and encrypt options
         //as well how many credits and what the current inventory is
-        
-        //buying equipment 
+
+        //buying equipment
         //increase amount in index amount if things are bought
         //decrease credits if things are bought
 
         //buying encrypt
-        //add pointer to encrypt linked list 
+        //add pointer to encrypt linked list
         //decrease credits if things are brought
 
         //selling equipment
-        //decrease amount in index amount if things are sold 
+        //decrease amount in index amount if things are sold
         //increase credit if things are sold
 
-
+        //round is complete
+        currentRound++;
     }
 
-    //save player score to leaderboard (prototype 2) 
+    //save player score to leaderboard (prototype 2)
 
-} 
+}
 
-     
+/******************************************************************************
+* Function:    briefing
+* Description: Brief the player on how to play the game
+* Parameters:  playerPtr, Player, pointer to the player object
+* Return:      void
+******************************************************************************/
+void briefing(Player *playerPtr){
+    char pause;
+
+    //end with printf for line-buffered console timely printing considerations
+    //80 characters is defacto standard output width for consoles
+    printf("/******************************************************************************\\\n");
+    printf("|================== Welcome to Cybersecurity Through History ==================|\n");
+    printf("\\******************************************************************************/\n");
+
+    //keep line spacing as own printf function for easier readability
+    printf("\n");
+    printf("Enter Player Name: ");
+    scanf("%s", &playerPtr->name);
+    //catch "phantom" enter
+    scanf("%c", &pause);
+
+    printf("\n");
+    printf("|================================== Briefing ==================================|\n");
+    printf("Hello %s, \n", playerPtr->name);
+    printf("\n");
+    printf("Your goal is to send as many confidential messages as possible throughout %d\n", NUM_ROUNDS);
+    printf("rounds. You have two controls: the amount of messages you send via EQUIPMENT,\n");
+    printf("and the amount of messages sent that remain confidential determined by the \n");
+    printf("ENCRYPTION METHOD used.\n");
+    //force player to press enter to continue
+    scanf("%c", &pause);
+    printf("At the end of each round, there is a chance that a new ATTACK will have been\n");
```

Comment:
```suggestion
    printf("\n");
    printf("At the end of each round, there is a chance that a new ATTACK will have been\n");
```
Adding another new line after user presses enter because of added [Press Enter] Message